lexicons/app/bsky/actor/defs.json

```diff
@@ -122,7 +122,28 @@
           "ref": "app.bsky.graph.defs#listViewBasic"
         },
         "following": { "type": "string", "format": "at-uri" },
-        "followedBy": { "type": "string", "format": "at-uri" }
+        "followedBy": { "type": "string", "format": "at-uri" },
+        "knownFollowers": {
+          "type": "ref",
+          "ref": "#knownFollowers"
+        }
+      }
+    },
+    "knownFollowers": {
+      "type": "object",
+      "description": "The subject's followers whom you also follow",
+      "required": ["count", "followers"],
+      "properties": {
+        "count": { "type": "integer" },
+        "followers": {
+          "type": "array",
+          "minLength": 1,
```

Comment:
I kinda want this to be `0`. To me an undefined response means something different from an empty array.
"I didn't try to load any knownFollowers on here" vs "I tried, but didn't find any"

thoughts?